1.data-exploration/sample_info_data_visualization.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+# import the necessary packages that will be utilized
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd 
+import pathlib
+import seaborn as sns
+
+# assign the desired file a variable using pathlib.Path command
+input_file = pathlib.Path("../0.data-download/data/sample_info.csv")
+
+# set the data frame to be the desired .csv file that is read by pandas(pd) using the pd.read_csv(desired file read as a previously defined variable)
+df = pd.read_csv(input_file)
+
+# print the parameters of the read file
+print(df.shape)
+
+
+# In[2]:
+
+
+# how many samples?
+n_samples = len(df["DepMap_ID"])
+print(f"Number of Samples: {n_samples} \n")
+
+# how many genes 
+all_cancers = df["age"].unique()
```

Comment:
this variable should be `all_ages`, correct?

And need to fix in the next line too :)

Reply:
this is correct i forgot to rename when i copy pasted the lines